usdf/storage.rst

```diff
@@ -8,40 +8,49 @@ Storage Locations
 =================
 
 Personal space:
- - Home directory space is available at /sdf/home/<first_letter_of_account>/<account> - standard S3DF personal allocation (25 GB)
- - Rubin-allocated space: /sdf/group/rubin/user/<account_name> with a 1 TB quota (a symlink to this directory, called rubin-user, is auto-created in your home directory). If you need more space, request it in #ops-usdf, and a ticket will get opened to request the quota increase.
+ - Home directory space is available at ``/sdf/home/<first_letter_of_account>/<account>`` - standard S3DF personal allocation (25 GB)
+ - Rubin-allocated space: ``/sdf/data/rubin/user/<account_name>`` with a 1 TB quota (a symlink to this directory, called ``rubin-user``, is auto-created in your home directory). If you need more space, request it in ``#ops-usdf``, and a ticket will get opened to request the quota increase.
  
 Web space:
- - "public_html" web access is available upon request: it will be visible at https://s3df.slac.stanford.edu/people/<user>
- - send email to usdf-help at slac.stanford.edu requesting a personal web directory
- - a public_html directory will be created in your home directory after acknowledging a usage policy
- - note that this is for static pages - no server-side content.
- - no symlinks outside public_html will be followed.
-
-Science data under /sdf/group/rubin/:
- - datasets/
- - lsstdata/offline/ (still in prep)
- - repo/
- - shared/data/test_data - daily updated checkouts of git lfs repos for use in testing code (includes the ap_verify CI datasets)
- - ncsa_home - copy of NCSA home directories (in prep, setting permissions)
- - ncsa_jhome - copy of NCSA RSP home directories (in prep, setting permissions)
+ - "public_html" web access is available upon request: it will be visible at ``https://s3df.slac.stanford.edu/people/<user>``
+ - send email to ``usdf-help`` at ``slac.stanford.edu`` requesting a personal web directory
+ - a ``public_html`` directory will be created in your home directory after acknowledging a usage policy
+ - note that this is for static pages - no server-side scripting.
+ - no symlinks outside ``public_html`` will be followed.
+
+Science data under ``/sdf/group/rubin/``:
+ - Much of this is still disorganized and contains vestiges of the structure from NCSA that was moved to USDF.  Bear with us until we get it all straightened out.
+ - ``shared/`` contains various datasets for common usage.
+   - ``shared/data-repos.yaml`` contains the list of aliases for Butler repos.
+   - shared/data contains daily-updated test and validation datasets.
```

Comment:
Should this be ``shared/data/test_data`` ?

Reply:
Both `test_data` and `validation_data` are updated.  (The other stuff needs to be moved away.)